imap_processing/quality_flags.py

```diff
@@ -0,0 +1,46 @@
+"""Bitwise flagging."""
+
+from enum import IntFlag
+
+
+class CommonFlags(IntFlag):
+    """Common quality flags."""
+
+    NONE = 0x0
+    INF = 2**0  # bit 0, Infinite value
+    NEG = 2**1  # bit 1, Negative value
+
+
+class ENAFlags(IntFlag):
+    """Common ENA flags."""
+
+    BADSPIN = 2**2  # bit 2, Bad spin
+
+
+class ImapUltraFlags(IntFlag):
```

Comment:
This is such a bummer that we can't inherit IntFlag enum using something like this without getting this error.
```
class ImapUltraFlags(CommonFlags, ENAFlags):
```
error:
```
E   TypeError: ENAFlags: cannot extend enumeration
```